app/views/wiki/show.html.erb

```diff
@@ -77,7 +76,7 @@
       <% if @node.power_tags('tabbed').include?("notes") %>
         <div class="tab-pane" id="tab-notes">
           <%= render :partial => "notes/notes" %>
-          <p><a href="/tag/<%= @node.slug %>"><%= raw t('wiki.show.more_research', :tag => @node.slug) %> &raquo;</a></p>
+          <p><a href="/tag/<%= @revision.drupal_node.slug_from_path %>"><%= raw t('wiki.show.more_research', :tag => @revision.drupal_node.slug_from_path) %> &raquo;</a></p>
```

Comment:
On this line in the admin controller test, `revision.drupal_node`:

https://github.com/publiclab/plots2/blob/master/test/functional/admin_controller_test.rb#L324

And on this line, `@revision.drupal_node`:

https://github.com/publiclab/plots2/blob/master/app/controllers/admin_controller.rb#L150

Yes, you can run just functional tests with `rake test:functionals`, units with `rake test:units`, integration with `rake test:integration`, and you can even run just one test file with:

`ruby -I test test/functional/notes_controller_test.rb` (for example)

I know it's time consuming to run them all!